main.go

```diff
@@ -215,7 +215,7 @@ func main() { // nolint:gocyclo
 	}
 
 	// Setup all Controllers
-	if err = toolchaincluster.NewReconciler(
+	if err = toolchainclustercache.NewReconciler(
 		mgr,
 		namespace,
 		memberClientTimeout,
```

Comment:
could you please update also the error message a few lines below:
https://github.com/codeready-toolchain/host-operator/blob/297336f840d09354906e791c521df9ebd3d579c3/main.go#L223

Reply:
you mean to add "ToolchainClusterCache" instead of "ToolchainCluster"..?